File: barcosEnContienda/src/gestorAplicacion/Personas/Tripulacion.java
package gestorAplicacion.Personas;
import gestorAplicacion.Lugares.*;
public class Tripulacion {
	private Capitanes capitan;
	public int cantidad_de_tripulantes;
	public int cantidad_maxima_de_tripulacion;
	private int ataque;
	private int defensa;
	private int efectividad_de_robo;
	private Lugar lugar;
	
	public Tripulacion(int cantidad_de_tripulantes, int cantidad_maxima_de_tripulacion, int ataque,
			int defensa, int efectividad_de_robo) {
		
		this.cantidad_de_tripulantes = cantidad_de_tripulantes;
		this.cantidad_maxima_de_tripulacion = cantidad_maxima_de_tripulacion;
		this.ataque = ataque;
		this.defensa = defensa;
		this.efectividad_de_robo = efectividad_de_robo;
		
	}
	
	public void vincularCapitan(Capitanes capitan){
		this.capitan=capitan;
	}
	public void vincularLugar (Lugar lugar) {
		this.lugar=lugar;
	}
}

File: practica-2-grupo-1-equipo-7/src/gestorAplicacion/personas/tripulacion.py
class Tripulacion:
    def __init__(self, cantidad_de_tripulantes, cantidad_maxima_de_tripulacion, ataque, defensa, efectividad_de_robo):
        self._cantidad_de_tripulantes = cantidad_de_tripulantes;
        self._cantidad_maxima_de_tripulacion = cantidad_maxima_de_tripulacion;
        self._ataque = ataque
        self._defensa = defensa
        self._efectividad_de_robo = efectividad_de_robo
        
    def vincularCapitan(self, capitan):
        self._capitan=capitan
    
    def vincularLugar(self, lugar):
        self._lugar=lugar